package.json

```diff
@@ -573,6 +577,11 @@
                     "when": "view == aws.explorer && viewItem == awsStateMachineNode",
                     "group": "1@1"
                 },
+                {
+                    "command": "aws.copyArn",
+                    "when": "view == aws.explorer && viewItem =~ /^(awsRegionFunctionNode|awsCloudFormationFunctionNode|awsStateMachineNode|awsCloudFormationNode|awsCloudFormationNode)$/",
```

Comment:
Why is awsCloudFormationNode in here twice? 
Does something in this list handle CloudFormation Stack nodes?

Reply:
Typo. `awsCloudFormationNode` represents a node for a CloudFormation Stack and should only be in here once.

Reply:
Typo. `awsCloudFormationNode` represents a node for a CloudFormation Stack and should only be in here once.